alicorn-expressions.lua

```diff
@@ -1290,7 +1290,7 @@ local top_level_block = metalanguage.reducer(
 			end
 			io.write(
 				line_setup_sequence
-					.. "processing "
+					.. "\rprocessing "
```

Comment:
add `io.flush()` after each "processing" `io.write` call (this one and the one above) to reduce jitter in the progress output due to buffering (`\r` isn't a newline therefore the output doesn't get automatically flushed)